### PR TITLE
Fixing sermon uploading

### PR DESCRIPTION
**Pull request by Bratalix**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,40 +8,45 @@
...
+    "speakingurl": "^14.0.1",
+    "theme-ui": "^0.3.5"
   },
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "eslint": "^6.3.0",
     "eslint-config-xo-react": "^0.20.0",
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-react-hooks": "^2.0.1",
+    "patch-package": "^6.2.2",
```
Comment (BarryThePenguin) [inline package.json:49]: This works, for now, but will be a little painful to update.. Hopefully sanity fixes the issue soon..

**Review by BarryThePenguin — APPROVED** [review]
Unsure what `plugins/s3-upload-widget` is, but it didn't look like crossroads was using it? It looks like it's leftover from some other work.. Might be ok to delete? Probably best to check with Alan first

File: package.json (hunk 1 above)
Comment (Bratalix) [reply package.json:49]: 👍

**Comment by Bratalix** [comment]
Yeah pretty sure it's a holdover from the old method but I'll ask Al

**Pull request merged**